index.html

```diff
@@ -4,14 +4,27 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" text="text/css" href="css/my_website.css">
     <title>Alejandro Ortiz</title>
 </head>
 
 <body>
-    <p>Hi, I’m @alejandro-ortiz <br>
-        I’m interested in Rocketry, Tennis, Web Development, Electronics <br>
+    <div class="main-header">
+        <h1 id="title">Alejandro Ortiz</h1>
+    </div>
+    <p class="text">Hi, I’m @alejandro-ortiz <br>
+        I’m interested in Rocketry, Tennis, Web Development, Electronics, and scouts<br>
         I’m currently learning Git and Javascript
     </p>
+    <div class="text2">
+        <p>My dream job is to work at apple by doing web development and helping with integrateing ideas.
+        </p>
+        <p>My dream goal is to help as many people understand electronics as I can and understand the components too.
+        </p>
+    </div>
+    <p class="text3">My favortie things to use are VSCode, Git, GitHub, MacOS, and over the ear headphones.
```

Comment:
What would be fun here is to show a picture of you in your element, and add some text showing you and pointing out your favorite tools / gear.

Reply:
I added a screenshot here but will add a real picture when I get home